HW_7/task_1.py

```diff
@@ -0,0 +1,32 @@
+# Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает, 
+# Вам стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое.
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+# Фразы отделяются друг от друга пробелами. 
+# Стихотворение  Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, 
+# если с ритмом все не в порядке
+
+word = input("Введите стихотворение: ")
+list = word.split()
+list1 = []
+vowels = "аеёиоуыэюя"
+for word in list:
+    count = 0
+    for letter in word:
+        if letter in vowels:
+            count = count + 1
```

Comment:
Один студент показал мне отличный лайфхак. Это гениально!
```suggestion
        count += letter in vowels
```
Подумайте внимательно, что тут происходит